src/styles/components/_search-bar.scss

```diff
@@ -142,6 +142,10 @@ $dso-search-bar-height: $u5;
   }
 }
 
+.dso-invalid .dso-search-bar-input input {
```

Comment:
`.dso-invalid` is een enkel woord (invalid) en geen component. Daarom moet de scoping strakker: `.dso-search-bar.dso-invalid`

Reply:
Waarom staat dit eigenlijk niet in het `.dso-search-bar` styling eiland?

Reply:
goede, aangepast